tests/test_app.py

```
@@ -0,0 +1,15 @@
+
+from requests import Response, head
+
+
+def test_app() -> None:
+    """
+    Test the reachability of Chattr.
+
+    Returns:
+        None
+    """
+    url = os.environ.get("CHATTR_URL", "http://localhost:7860/")
```

Ruff (0.12.2)

Undefined name `os`

(F821)